pkg/network/usm/shared_libraries_test.go

```diff
@@ -486,14 +491,14 @@ func initEBPFProgram(t *testing.T) *ddebpf.PerfHandler {
 		Probes: []*manager.Probe{
 			{
 				ProbeIdentificationPair: manager.ProbeIdentificationPair{
-					EBPFFuncName: "kprobe__" + probe,
+					EBPFFuncName: fmt.Sprintf("tracepoint__syscalls__sys_enter_%s", openatSysCall),
```

Comment:
I think it makes sense to extract a common function for the enter/exit `fmt.Sprintf`.